glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -60,9 +60,9 @@ internal class OnGleanEventsImpl(
         PingUploadWorker.enqueueWorker(glean.applicationContext)
     }
 
-    override fun startMetricsPingScheduler() {
+    override fun startMetricsPingScheduler(): Boolean {
         glean.metricsPingScheduler = MetricsPingScheduler(glean.applicationContext, glean.buildInfo)
-        glean.metricsPingScheduler?.schedule()
+        return glean.metricsPingScheduler!!.schedule()
```

Comment:
I guess it is better to explode here than to fail silently with the `?`

Reply:
We set the value in the line above, so `!!` can't fail (unless the java VM breaks)